frontend/src/components/application/QualityModerationSlideOver.tsx

```diff
@@ -255,6 +256,13 @@ const QualityCategories = ({
     query.data.data.guidelines.map((guideline) => guideline.guideline.category),
   )
 
+  const issueLink = newGithubIssueUrl({
+    title: `Quality guideline problems for ${app.name}`,
+    user: "flathub",
+    repo: app.id,
+    body: `Please list the problems you want help with in this issue:\n\n\n\n\n\n\n\n CC: @flathub/quality-moderation (please don't delete this)`,
```

Comment:
This won't work outsiders can't mention or see teams of an org unless they are members.

Reply:
I suggest to either reuse the main Flathub/flathub repo or have a seperate repo under the flathub org for issues.

We can have some automation to tag the team and label issues etc.

Reply:
Settled for using flathub/flathub with some bot automation